src/components/structures/MatrixChat.js

```diff
@@ -1052,8 +1055,7 @@ module.exports = React.createClass({
                     page_element = (
                         <RoomView
                             ref="roomView"
-                            roomId={this.state.currentRoom}
-                            roomAlias={this.state.currentRoomAlias}
+                            roomAlias={this.state.currentRoom || this.state.currentRoomAlias}
```

Comment:
Isn't this setting the room alias prop to a room ID? Wouldn't currentRoom always be present if currentRoomAlias is?